src/diffusers/pipelines/pndm/pipeline_pndm.py

```diff
@@ -43,14 +43,14 @@ def __call__(self, batch_size=1, generator=None, torch_device=None, num_inferenc
         )
         image = image.to(torch_device)
 
-        prk_time_steps = self.scheduler.get_prk_time_steps(num_inference_steps)
+        prk_time_steps = self.scheduler.get_timesteps(num_inference_steps, step_mode="prk")
         for t in tqdm(range(len(prk_time_steps))):
```

Comment:
this should be:

```
for t in tqdm(self.prk_timesteps):
...
```

IMO

and it'd be nice if we would only have:

```
scheduler.set_timesteps(...)
```

which sets up both `self.prk_timesteps` and `self.plms_timesteps` arrays

Reply:
Yeah I like this, going to make these changes now.